src/app/components/gallery-elements-bar/gallery-elements-bar.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="mx-4 mt-2 pb-1 border-bottom bd-highlight border-dark d-flex justify-content-between">
+
+    <div class="p-2 mx-3 bd-highlight">
+        <input type="search" class="form-control form-control-dark" placeholder="Search..." aria-label="Search">
+    </div>
+
+    <div *ngIf="isAvaliable" class="p-2 me-auto bd-highlight">
+        <app-select></app-select>
```

Comment:
Please review (throw all pr) is it possible to replace such cases to <app-select />.

Reply:
No way to do this.